<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="appNameEdit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="formatsEdit"/>
    </item>
    <item>
     <widget class="QPushButton" name="registerButton">
      <property name="text">
       <string>Register Application</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="filePathEdit"/>
    </item>
    <item>
     <widget class="QPushButton" name="openButton">
      <property name="text">
       <string>Open File</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="showAppsButton">
      <property name="text">
       <string>Show Registered Applications</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="showFormatsButton">
      <property name="text">
       <string>Show Registered Formats</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="textBrowser"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
